src/prefect/cli/global_concurrency_limit.py

```diff
@@ -53,3 +65,64 @@ async def list_global_concurrency_limits():
         )
 
     app.console.print(table)
+
+
+@global_concurrency_limit_app.command("inspect")
+async def inspect_global_concurrency_limit(
+    name: str,
+    output: Optional[OutputFormat] = typer.Option(
+        None,
+        "--output",
+        "-o",
+        help="Output format for the command.",
+        case_sensitive=False,
+    ),
+    file_path: Optional[Path] = typer.Option(
+        None,
+        "--file",
+        "-f",
+        help="Path to .py file containing block types to be registered",
+    ),
+):
+    """
+    Inspect a global concurrency limit.
+
+    Arguments:
+        name: str: [required]
+            - The name of the global concurrency limit to inspect.
+        output: Optional[OutputFormat]: [optional]
```

Comment:
We should probably pick one way of saying that the argument is optional. I have a _very_ slight preference for `Path: [optional]` simply because of the `[required]` above. Alternatively we could remove `[required]` and let the type hint speak to the fact the others are optional.

Reply:
Ah yeah, I agree, that was an oversight. Good call. Applied in [9360e74](https://github.com/PrefectHQ/prefect/pull/13196/commits/9360e74ebad6266fe3cfc0ba16f7016e3c1d7eb3)

Reply:
And [a7c98a3](https://github.com/PrefectHQ/prefect/pull/13196/commits/a7c98a349d9ad26bc0be05d554621885327bc6b3)